utils/autotailor

```diff
@@ -198,27 +247,66 @@ class Tailoring:
         title.text = self.profile_title
 
         self._add_rule_select_operations(profile)
-        self._add_value_selections(profile)
+        self._add_value_overrides(profile)
         self.rule_refinements_to_xml(profile)
+        self.value_refinements_to_xml(profile)
 
         root_str = ET.tostring(root)
         pretty_xml = xml.dom.minidom.parseString(root_str).toprettyxml()
         with open(location, "w") if location != "-" else sys.stdout as f:
             f.write(pretty_xml)
 
+    def import_json_tailoring(self, json_tailoring):
+        import json
+        with open(json_tailoring, "r") as jf:
+            all_tailorings = json.load(jf)
+
+        if 'profiles' in all_tailorings and all_tailorings['profiles']:
+            # We currently support tailoring of one profile only
```

Comment:
Can we check and raise if there are multiple profiles?

Reply:
Sure.